capstone/heroku.md

```diff
@@ -10,42 +10,88 @@ $ brew tap heroku/brew && brew install heroku
 $ heroku --version
 ```
 
-3. Now we can login to Heroku from the command line. Enter your email and password for your group Heroku account.
+3. First we'll use Heroku's browser authenticated login in via the command line
+  - 3a. Login to the Heroku website with your groups Heroku credentials.  
+  - 3b. Run the following command 
 ```bash
-$ heroku login -i
+$ heroku login 
 ```
+  - 3c. When the new chrome page opens click the button that says "log in"
 
-4. Create a new app on Heroku. The url will be printed in the command line.
-```bash
-$ heroku create
+4. Once you are logged in find the button that says "create a new app" on the Heroku dashboard. 
+  - 4a. Name your app something similar to your capstone project. your app name will also be the name of your url
+  - 4b. The new app will have a a set of cli prompts. Run the prompts most similar to the following  
+```
+$ heroku git:remote -a <your-app-name>
+$ git commit -am "Making things better"
 ```
+  - 4c. Your project is now connected to heroku 
 
-5. Ensure that the remote git repo on Heroku was added to your local git config.
-```bash
-$ git config --list | grep heroku
+## Now we need to install some deployment tools and configurations
+
+5.  Run the following command to allow builds of your rails app to happen on intel chip based computers so the build can be sent to Heroku. 
+```
+$ bundle lock --add-platform x86_64-linux
+$ git add .
+$ git commit -m "added x86_64 linux to platforms" 
+```
+
+6. Update the bundle packages to support this platform adjustment
+```
+$ bundle update
+```
+
+7. If you have the master.key for the project in your config folder file skip this step.
+  - 7a. To create a new master.key and credential.yml file run the command
```

Comment:
When creating a new master.key, the credential.yml file needs to be deleted from the app.